app/javascript/packages/webauthn/webauthn-input-element.spec.ts

```diff
@@ -85,4 +99,30 @@ describe('WebauthnInputElement', () => {
       });
     });
   });
+
+  context('Desktop F/T unlock A/B test', () => {
```

Comment:
How is this different from what we're testing above? (L43)

Reply:
I tested both disabled and enabled here and by doing this, I made a duplicate. Since 43 is a more concise test as well it looking like that I don't need a `is hidden` test, I can remove the below tests.